Taarafo.Portal.Web.Tests.Unit/Services/PostViews/PostViewServiceTests.Exceptions.RetrieveAll.cs

```diff
@@ -0,0 +1,89 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Moq;
+using Taarafo.Portal.Web.Models.PostViews;
+using Taarafo.Portal.Web.Models.PostViews.Exceptions;
+using Xunit;
+
+namespace Taarafo.Portal.Web.Tests.Unit.Services.PostViews
+{
+    public partial class PostViewServiceTests
+    {
+        [Theory]
+        [MemberData(nameof(DependencyExceptions))]
+        public async Task ShouldThrowPostViewDependencyExceptionIfDependecyErrorOccursAndLogItAsync(
```

Comment:
We need not repeat the entity again 'PostViewDependencyException'. It can be 'ShouldThrowDependencyException'